src/client.toit

```diff
@@ -9,6 +9,25 @@ import .transport
 import .packets
 import .topic_filter
 
+class SendTrackingTransport_ implements Transport:
```

Comment:
This feels a bit like overkill to me. I'd probably just add a send helper on the Client class.

Reply:
done.